### DISCUSSION TITLE
'Document' object has no attribute '_getXrefLength' or 'extract_image'

### DISCUSSION THREAD

**Discussion by bywyu** [pymupdf/PyMuPDF#1166] [Looking for help]
I try to run

https://github.com/pymupdf/PyMuPDF-Utilities/blob/master/examples/extract-imgb.py

'Document' object has no attribute '_getXrefLength' or 'extract_image'

if MuPDF version is 1.18.0,get this error 'Document' object has no attribute 'extract_image' ,if version 1.18.6 will get 'Document' object has no attribute  'extract_image'

how to run this file ?

**Comment by JorjMcKie** [comment]
I do not accept issues for scripts in this repo of examples.
Please upgrade to v1.18.15 and change `_getXrefLength` to `xref_length`.
Then please submit a PR with the corrected script.
Thank you!
